### PR TITLE
Add more e-commerce sites

### PR DESCRIPTION
**Pull request by phlinhng**

### CODE REVIEW

File: data/pchome [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ecimg.tw
+full:ecapi-pchome.cdn.hinet.net
+pchome.com.tw
+pcloud.tw
```
Comment (DuckSoft) [inline data/pchome:4]: I cannot resolve `pcloud.tw` on my Hong Kong VPS.
Is that correct?

**Review by DuckSoft — COMMENTED** [review]
LGTM except that one.

File: data/pchome (hunk 1 above)
Comment (phlinhng) [reply data/pchome:4]: try `apid.pcloud.tw` instead. However, both `pcloud.tw` and `apid.pcloud.tw` are resolve-able with my local DNS and HK VPS.

**Comment by Loyalsoldier** [comment]
I add them all to `category-ecommerce` list.

**Pull request merged**